### PR TITLE
add namespace in client options

### PR DESCRIPTION
**Pull request by zklgame**

### CODE REVIEW

File: src/main/java/io/xdb/core/client/Client.java [hunk 1]
```diff
@@ -129,7 +128,11 @@ private String startProcessInternal(final String processType, final String proce
             : process.getOptions();
 
         final ProcessExecutionStartRequest request = new ProcessExecutionStartRequest()
-            .namespace(processOptions.getNamespace())
+            .namespace(
+                Strings.isNullOrEmpty(processOptions.getNamespace())
```
Comment (longquanzheng) [inline src/main/java/io/xdb/core/client/Client.java:132]: I feel like it's confusing to have two places defining the namespace. 
WDYT if we remove the one in processOptions?

**Review by longquanzheng — APPROVED** [review]
lgtm with a suggestion

File: src/main/java/io/xdb/core/client/Client.java (hunk 1 above)
Comment (zklgame) [reply src/main/java/io/xdb/core/client/Client.java:132]: Makes sense. Then maybe we should also remove namespace from stopProcess/describeProcess?

Comment (longquanzheng) [reply src/main/java/io/xdb/core/client/Client.java:132]: Yeah good point. Agreed

**Pull request merged**